prometheus_alerting_consistency.md

```diff
@@ -0,0 +1,103 @@
+# Prometheus Alerting Consistency
+
+## Summary
+
+This guide is originally based on the [Alerting Consistency][1] proposal for OpenShift alerts.
+
+Clear and actionable alerts are a key component of a smooth operational experience.
```

Comment:
I think there is a need to define the term `actionable` here as it can be interpreted differently by different teams/users. Such as does actionable means it has a SOP, or what are the basic requirements to determine that an alert is actionable?

Reply:
Agreed. The general principle is that there should be something for an SRE to "do" to resolve the issue. That action may be following a SOP, but the SOP must contain actions that are achievable and specific. It is not sufficient to say: "logs at the logs and determine the next action". It should say something like: "look at the logs and if it contains this error then run this command". If an SRE can do something then it shouldn't be an alert.

Reply:
```suggestion
Clear and actionable alerts are a key component of a smooth operational experience. If an alert doesn't have a documented procedure to follow when it fires, it can lead to an increase in mean time to repair (MTTR). 
```
@pepedocs @brizrobbo Does this sound reasonable? Feel free to suggest alternate wording.
I think what you 're seeing is the ideal scenario and what should be strived for. 
In reality, I think we all know that's it's less clear cut and there's going to be an element of troubleshooting with some alerts (otherwise why would you have an alert with a clear fix documented rather than get the problem fixed or automate it if it's toil).

Reply:
Great @david-martin .... I'm nit-picking, but what about adding the word "proven" in there  like this: ...a **proven** documented procedure..."? Not necessary, but I was just trying to capture the idea that the process must be tested.

Reply:
@david-martin :+1: